action.yml

```diff
@@ -47,6 +47,9 @@ inputs:
   mavenReleaseArguments:  # id of input
     description: 'Arguments for Maven release'
     required: false
+  releaseArguments:
```

Comment:
Some better name? Very easy to confuse with the existing arg.

The new one could be `mavenReleaseArguments` and the old one renamed to `mavenReleasePrepareArguments`.

Reply:
The old `mavenReleaseArguments` after all is only passed to `release:prepare` - not to `release:perform` so it is misleading already.

Reply:
how about `releasePluginArguments` and keeping the old one as is? The release plugin passes `-Darguments` down to other maven executions.